flink-end-to-end-tests/flink-glue-schema-registry-avro-test/src/test/java/org/apache/flink/glue/schema/registry/test/GlueSchemaRegistryAvroKinesisITCase.java

```diff
@@ -125,25 +133,18 @@ public void testGSRAvroGenericFormatWithFlink() throws Exception {
         }
         log.info("results: {}", results);
 
-        Assert.assertEquals(
-                "Results received from '" + OUTPUT_STREAM + "': " + results,
-                messages.size(),
-                results.size());
-        Assert.assertTrue(messages.containsAll(results));
+        assertThat(results).isEqualTo(messages);
```

Comment:
Is order required for assertion? It seems to be ignored previously

Reply:
Yes, good point since Kinesis does not guarantee ordering 👍